projects/plugins/jetpack/extensions/blocks/contact-form/components/jetpack-field-controls.js

```diff
@@ -18,13 +21,86 @@ import JetpackFieldWidth from './jetpack-field-width';
 import JetpackManageResponsesSettings from './jetpack-manage-responses-settings';
 
 const JetpackFieldControls = ( {
-	setAttributes,
-	width,
+	attributes,
 	id,
-	required,
 	placeholder,
 	placeholderField = 'placeholder',
+	required,
+	setAttributes,
+	width,
+	type,
 } ) => {
+	const setNumberAttribute = ( key, parse = parseInt ) => value => {
+		const parsedValue = parse( value, 10 );
+
+		setAttributes( {
+			[ key ]: ! isNaN( parsedValue ) ? parsedValue : '',
+		} );
+	};
+
+	let colorSettings = [];
+	let borderWidthControls = '';
+	let borderRadiusControls = '';
+
+	if ( typeof type === 'string' && ( type === 'radio' || type === 'checkbox' ) ) {
+		colorSettings = [
+			{
+				value: attributes.labelColor,
+				onChange: value => setAttributes( { labelColor: value } ),
+				label: __( 'Label text', 'jetpack' ),
+			},
+			{
+				value: attributes.inputColor,
+				onChange: value => setAttributes( { inputColor: value } ),
+				label: __( 'Field text', 'jetpack' ),
+			},
+		];
+		borderWidthControls = '';
+		borderRadiusControls = '';
+	} else {
+		borderWidthControls = (
```

Comment:
I think this would read a lot better if we didn't use variables for `borderWidthControls` and `borderRadiusControls` and instead you can just add the condition right in the JSX:

```
{ hasBorderControls && (
    <TextControl
        ...
    />
) }
```

Reply:
Thanks, that worked perfectly.